module/VuFind/src/VuFindTest/Integration/MinkTestCase.php

```diff
@@ -246,12 +246,28 @@ protected function changeConfigFile($configName, $settings, $replace = false)
 
             $this->modifiedConfigs[] = $configName;
         }
+        $this->writeConfigFile($local, $settings, $replace);
+    }
+
+    /**
+     * Write settings to a file.
+     *
+     * @param string $path     Path of file to modify.
+     * @param array  $settings Settings to change.
+     * @param bool   $replace  Should we replace the existing config entirely
+     *                         (as opposed to extending it with new
+     *                         settings)?
+     *
+     * @return void
+     */
+    protected function writeConfigFile(string $path, array $settings, bool $replace = false)
```

Comment:
Adding return value here too.

```suggestion
    protected function writeConfigFile(string $path, array $settings, bool $replace = false): void
```